python/llama-index-server/llama_index/server/api/utils/workflow.py

```
@@ -0,0 +1,115 @@
+from llama_index.server.utils.class_meta_serialization import (
+    type_from_identifier,
+    type_identifier,
+)
+
+
+class HITLWorkflowService:
+    # A key in context that stores the HITL event type
+    HITL_CONTEXT_KEY = "human_response_type"
+
+    @staticmethod
+    def get_storage_path(chat_id: str) -> Path:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)